src/Config/Environment.php

```diff
@@ -18,7 +18,7 @@ class Environment
      *   master - is for integration environment;
      *   production and staging are for production and staging environments respectively.
      */
-    private const GIT_MASTER_BRANCH_RE = '/^(master|production|staging)(?:-[a-z0-9]+)?$/i';
+    private const GIT_MASTER_BRANCH_RE = '/^(master|production|staging)(?:[a-z0-9-]+)?$/i';
```

Comment:
Looks like we can left only `'/^(master|production|staging)/i'` as `(?:[a-z0-9-]+)?` contains all possible symbols in the branch name.

Reply:
Agree, fixed